### DISCUSSION TITLE
Missing documentation on GitHub Actions badges

### DISCUSSION THREAD

**Discussion by SamChou19815** [community/community#27129] [Actions]
<p>I see some GitHub repos and many other repos are already using the GitHub actions badges.</p>
<p>Example: </p>
<pre><code>https://github.com/actions/toolkit/workflows/Main%20workflow/badge.svg
</code></pre>
<p>is used in <a href="https://github.com/actions/toolkit" rel="nofollow noopener">https://github.com/actions/toolkit</a>’s readme.</p>
<p>However, I can’t find any <a href="https://help.github.com/en/categories/automating-your-workflow-with-github-actions" rel="nofollow noopener">official docs</a> on that topic. It would be nice if it’s better documented.</p>

**Comment by lannonbr** [comment, marked as answer]
<p>There’s docs on it now here: <a href="https://help.github.com/en/articles/configuring-a-workflow#adding-a-workflow-status-badge-to-your-repository" rel="nofollow noopener">https://help.github.com/en/articles/configuring-a-workflow#adding-a-workflow-status-badge-to-your-repository</a></p>

**Comment by randy3k** [comment]
<p>What is the best url pointed by the badge?</p>
<p>There is </p>
<pre><code>https://github.com/&lt;username&gt;/&lt;repo&gt;/commit/&lt;sha&gt;/checks
</code></pre>
<p> but it seems that <code>\&lt;sha\&gt;</code> cannot be <code>master</code> which defects this usage.</p>

**Comment by phil-lgr-nr** [comment]
<p>What about private repos?</p>

**Comment by 4lejandrito** [comment]
<p>Hi <a class="mention" href="/randy3k">@randy3k</a>,</p>
<p>This is the closest I got for my repo:</p>
<pre><code>https://github.com/4lejandrito/creepyface/actions?query=workflow%3ABuild+branch%3Amaster
</code></pre>
<p>It links to the actions view filtering by workflow name and branch.</p>
<p>This is the code I use in the README to show the badge:</p>
<pre><code>[![Build](https://github.com/4lejandrito/creepyface/workflows/Build/badge.svg)](https://github.com/4lejandrito/creepyface/actions?query=workflow%3ABuild+branch%3Amaster) 
</code></pre>
<p><strong>Edit:</strong> I just found out that this URL is not public, only registered users can view it. Unfortunately there seems to be nowhere to link from the badge for everyone to see.</p>

**Comment by rowi1de** [comment]
<p>Is there a GitHub Action up easily to add a new event type/status?<br>
I would like to show a build number, they are generated in the workflow.</p>
<p>I included a “deployment” / “deployment_status” Badge and update it via the Action but the Badge doesn’t update</p>

**Comment by shonfeder** [comment]
<p>This link doesn’t point to documentation on badges any more, afaict. Maybe this is the right link now? <a href="https://docs.github.com/en/actions/managing-workflow-runs/adding-a-workflow-status-badge" rel="noopener nofollow ugc">https://docs.github.com/en/actions/managing-workflow-runs/adding-a-workflow-status-badge</a></p>

**Comment by LorenzoBettini** [comment]
<p>If I’m not wrong, the documentation doesn’t say anything about the fact that the badge can be created by the web interface: navigate to a workflow, click on the “…” and select “Create status badge”.</p>
